src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPasswordConfirmation.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@{
+    ViewLayout = "Layout__Login";
+}
+
+<h1>@T["Reset Password"]</h1>
+<p>
+    @T["Your password has been reset. Melon"]
```

Comment:
Melon?

Reply:
upps sorry this is a test I made cause I was not able to override this cshtml I forgot to undo those changes